src/pool-bin/libraries/math/PackedUint128Math.sol

```diff
@@ -209,17 +212,20 @@ library PackedUint128Math {
 
     /// @dev given amount and protocolFee, calculate and return external protocol fee amt
     /// @param amount encoded bytes with (x, y)
-    /// @param fee protocolFee
-    function getExternalFeeAmt(bytes32 amount, uint16 fee) internal pure returns (bytes32 z) {
-        if (fee == 0) return 0;
+    /// @param protocolFee protocolFee
```

Comment:
missing `/// @param swapFee` maybe can just copy other places on what protocolFee / swapFee is over here